### DISCUSSION TITLE
How to install Tailwind in a fresh Quasar CLI with Vite app

### DISCUSSION THREAD

**Discussion by Rolanddoda** [quasarframework/quasar#12820] [General - Components / Directives / etc]
I managed to get TailwindCSS working with Quasar CLI - webpack version but now with the Quasar CLi - vite Tailwind doesn't work. I followed the steps here -> https://tailwindcss.com/docs/guides/vite but it doesn't work and it throws errors like this:

![image](https://user-images.githubusercontent.com/18482346/158596580-4f49044b-340c-441f-9133-4f72dc789e49.png)

**Comment by yusufkandemir** [comment]
🤷 Without a repro we can't know what went wrong with your project. I just created a new project and I didn't get any errors. I can't guarantee if there will not be any conflicts or something, but this would be the way to install it:

1. Create a new Quasar CLI app(`yarn create quasar`), select v2, Vite, SCSS, the rest depends on personal preference and project requirements.
2. Install `yarn add -D tailwindcss postcss autoprefixer`
3. Run `yarn tailwindcss init`. _The `-p` flag in the tutorial doesn't do anything as the `postcss.config.js` file already exists. The tutorial you've sent doesn't mention this case, as it's targeted specifically to `create-vite`_.
4. Update `tailwind.config.js`:
```js
content: [
  './index.html',
  './src/**/*.{vue,js,ts}',
],
```
5. Update `postcss.config.js`.
```js
plugins: [
  require('tailwindcss'),

  // ...
```
6. Update `src/css/app.scss`:
```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```
7. Run the dev server: `quasar dev` / `yarn quasar dev`
